widget/embedded/src/components/Route/index.ts

```diff
@@ -0,0 +1 @@
+export { Route } from './Route';
```

Comment:
I think it's technically a `container` so we can move it to `/src/containers/Route`.